SPECS/moby-engine/moby-engine.spec

```diff
@@ -126,6 +126,9 @@ fi
 %{_unitdir}/*
 
 %changelog
+* Sat Aug 08 2023 Muhammad Falak <mwani@microsoft.com> - 20.10.25-1
```

Comment:
please change to Tuesday, Aug 08, or perhaps Tue Aug 15

Reply:
Ahha :( `copy-pasta` :(
Good catch